src/components/DataView/DataViewDisplayGrid/DataViewDisplayGrid.styled.ts

```diff
@@ -0,0 +1,139 @@
+import styled from "styled-components";
+import theme, { BREAKPOINTS } from "@root/theme/theme";
+
+const BIG_SCREEN_BREAKPOINT = BREAKPOINTS.topComponent.bigScreenView + "px";
+const RESPONSIVE_BREAKPOINT = BREAKPOINTS.topComponent.responsiveView + "px";
+const MOBILE_BREAKPOINT = BREAKPOINTS.mobile + "px";
+
+export const StyledDiv = styled.div`
+	& > .topRowBulkAll {
+		margin-bottom: 5px;
+	}
+
+	& > .grid {
+		display: grid;
+		grid-template-columns: repeat(1, 1fr);
+		grid-column-gap: 16px;
+		grid-row-gap: 24px;
+
+		@media (min-width: ${MOBILE_BREAKPOINT}) {
+			grid-template-columns: repeat(2, 1fr);
+		};
+
+		@media (min-width: 768px) {
+			grid-template-columns: repeat(3, 1fr);
+		};
+
+		@media (min-width: ${RESPONSIVE_BREAKPOINT}) {
+			grid-template-columns: repeat(4, 1fr);
+		};
+
+		@media (min-width: 1296px) {
+			grid-template-columns: repeat(5, 1fr);
+		};
+
+		@media (min-width: ${BIG_SCREEN_BREAKPOINT}) {
+			grid-template-columns: repeat(6, 1fr);
+		}
+	}
+
+	& > .grid > .cell {
+		min-width: 0;
+	}
+
+	& > .grid > .cell.checked > .image > .checkboxContainer {
+		opacity: 1;
+	}
+
+	& > .grid > .cell > .image img {
+		max-width: 100%;
+		min-width: 100%;
+	}
+
+	& > .grid > .cell h2 {
+		color: ${theme.newColors.almostBlack["100"]};
+		margin: 8px 0 8px 0;
+		max-width: 161px;
+		font-size: 13px;
+		font-weight: 590;
```

Comment:
I know figma has 590 but let's use our semibold which is 600, the 590 doesn't exist in the "official" weights. I'll talk with Fa about this